libs/blocks/quiz-entry/quiz-entry.js

```diff
@@ -4,6 +4,12 @@ import { mlField, getMLResults } from './mlField.js';
 import { GetQuizOption } from './quizoption.js';
 import { quizPopover, getSuggestions } from './quizPopover.js';
 
+export const locationWrapper = {
+  redirect: (url) => {
+    window.location = url;
```

Comment:
Have you considered [assign](https://developer.mozilla.org/en-US/docs/Web/API/Location/assign)? It may be useful to navigate back, if that's desired.